demo-client-application/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "demo-client-application",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@lens-protocol/metadata": "^1.0.5",
+    "@lens-protocol/react-web": "^2.0.0-alpha.19",
+    "@lens-protocol/wagmi": "^3.0.0-alpha.22",
+    "@lens-protocol/widgets-react": "^2.0.7",
+    "@radix-ui/react-dropdown-menu": "^2.0.5",
+    "@radix-ui/react-slot": "^1.0.2",
+    "@types/node": "20.6.0",
+    "@types/react": "18.2.21",
+    "@types/react-dom": "18.2.7",
+    "connectkit": "1.6.0",
+    "autoprefixer": "10.4.15",
+    "class-variance-authority": "^0.7.0",
+    "clsx": "^2.0.0",
+    "eslint": "8.49.0",
+    "eslint-config-next": "13.4.19",
+    "ethers": "^5.7.2",
+    "lucide-react": "^0.276.0",
+    "next": "13.4.19",
+    "next-themes": "^0.2.1",
+    "postcss": "8.4.29",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "tailwind-merge": "^1.14.0",
+    "tailwindcss": "3.3.3",
+    "tailwindcss-animate": "^1.0.7",
+    "typescript": "5.2.2",
+    "viem": "^1.15.0",
+    "wagmi": "^1.4.12"
```

Comment:
with latest version of the lens react sdk you can use latest wagmi and view (v2)